pkg/features/kube_features.go

```diff
@@ -851,21 +787,13 @@ var defaultKubernetesFeatureGates = map[featuregate.Feature]featuregate.FeatureS
 
 	CPUCFSQuotaPeriod: {Default: false, PreRelease: featuregate.Alpha},
 
-	CPUManager: {Default: true, PreRelease: featuregate.GA, LockToDefault: true}, // GA in 1.26
```

Comment:
/cc @ffromani 
Should this be removed in v1.27 or later?

Reply:
CPUManager is not removed completely if needed. 
/lgtm cancel

Reply:
I think this should be removed anytime starting 1.27, yes. There are other FG referencing CPUManager, but those are indpeendent and it's fine to leave them there.

Reply:
![image](https://user-images.githubusercontent.com/2010320/212293251-e9e5729e-29b5-4ea7-86d0-11364465d0cb.png)
Here should be removed as well.


Sorry, miss that. @cyclinder  would you update it?.

Reply:
/unhold 
for CPUManager part update

Reply:
policy asks to keep the feature gate for 2 releases. See https://kubernetes.io/docs/reference/using-api/deprecation-policy/#deprecating-a-feature-or-behavior